packages/openapi3/src/openapi.ts

```diff
@@ -1643,11 +1643,20 @@ function createOAPIEmitter(
     return callSchemaEmitter(type, visibility) as any;
   }
 
-  function attachExtensions(program: Program, type: Type, emitObject: any) {
+  function attachExtensions(
+    program: Program,
+    type: Type,
+    emitObject: any,
+    enabled: boolean = false,
```

Comment:
suggest to add new method: `attachSchemaExtensions` for adding `["minProperties", "maxProperties", "uniqueItems", "multipleOf"]` only